syntax = "proto3";
message Image {
    string image_name = 1;
}
service AnimeStyle {
    rpc ConvertAnimeStyle (Image) returns (Image) {}
}